std/http/file_server_test.ts

```diff
@@ -79,3 +79,23 @@ test(async function serveFallback(): Promise<void> {
     killFileServer();
   }
 });
+
+test(async function servePermissionDenied(): Promise<void> {
+  const deniedServer = Deno.run({
+    args: [Deno.execPath(), "run", "--allow-net", "http/file_server.ts"],
+    stdout: "null",
+    stderr: "piped"
+  });
+
+  const curl = Deno.run({
+    args: ["curl", "http://localhost:4500/"],
+    stdout: "null"
+  });
```

Comment:
I don't think the tests should depend on curl. Can you use fetch instead?

Reply:
@ry done.